Rota-Process.md

```diff
@@ -0,0 +1,302 @@
+>   applied to them.
+
+##### Issue triage
+
+Each new issue needs to be read carefully. An issue is an arbitrary
+description of a problem or request so to allow the issues to be categorized
+and searched for, we try to add meaningful labels to each one.
+
+If you are unsure of which labels to apply to an issue, talk to the rest of
+the [Rota team](#talk-to-each-other) or just ask
+[on Slack/IRC](https://github.com/kata-containers/community#join-us).
+
+Here is some general advice:
+
+- Security issue
+
+  If the issue is reporting a security issue, add the `security` tag and
+  [immediately contact the team](https://github.com/kata-containers/community#join-us)
+  since it may be necessary to follow the
+  [vulnerability Management Process](https://github.com/kata-containers/community/blob/master/VMT/VMT.md).
+
+- "Crasher bug"
+
+  If the issue is reporting a crash or catastrophic error scenario add or more
+  one of these labels (whichever is most appropriate):
+
+  - `crash`
+  - `data-loss`
+  - `hang`
+  - `resource-leak`
+  - `unreliable`
+
+  If you add one of these labels, please also
+  [contact the team](https://github.com/kata-containers/community#join-us)
+  so they are aware of the issue as soon as possible.
+
+- Top priority
+
+  If an issue is extremely urgent and should be fixed as soon as possible, you
+  should add the `highest-priority` label to flag this to the entire team.
+
+- Release gating
+
+  If you think the issue should be fixed before the next release, add the
+  `release-gating` label.
+
+- Important medium / long term task
+
+  If an issue is an very important but is not time-critical, you should add
+  the `highest-severity` labels.
+
+- Simple task for new contributors
+
+  If you think the issue is a simple, relatively quick to resolve and
+  "self-contained", you should add the `good-first-issue` issue to mark it as
+  being suitable for a new contributor to look at.
+
+- Invalid
+
+  If you think the issue is somehow not appropriate, add the `invalid` label.
+
+- Duplicate
+
+  If the issue is a duplicate of an existing issue you've already seen, add
+  the `duplicate` label along with a comment referencing the original.
+
+- Limitation
+
+  If the issue describes a system limitation, add the `limitation` label.
+
+  If the issue is a
+  [known limitation](https://github.com/kata-containers/documentation/blob/master/Limitations.md),
+  close the issue with a comment referencing the original limitation issue
+  number so the issue author understands why the issue was closed.
+
+- Teams
+
+  If you know which team the issue should be assigned to, add one of the
+  `team/*` labels.
+
+- Related project
+
+  If the issue relates to another project, add one of the `related/*`
+  labels.
+
+- Issue is lacking something
+
+  If you think the issue is incomplete somehow, consider adding one of the
+  `needs-*` labels.
+
+##### General process
+
+BUG: FIXME: link to issue reports in wiki.
```

Comment:
Hard to tell, is this you saying there is a `BUG: FIXME:` in this PR/doc that needs fixing before we land it?

Reply:
That should have been removed - fixed.